docs/source/portal-molecule.rst

```diff
@@ -0,0 +1,84 @@
+Portal Molecule
+===============
+
+The Portal Molecule class can represent many different quantum chemical molecules and also provides the primary manipulation tool for molecules.
+
+Creating Molecules
+------------------
+
+Molecules can be created via NumPy, Psi4-like molecule strings, and QCSchema JSON representations.
```

Comment:
"QCJson_Schema" is another thing that needs a `|qcjson|` alias. I can see its official name changing.